presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,26 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that the Presto process runs as. Currently, 
+  Presto requires a minimum of 4096 open file descriptors, and strongly 
+  recommends at least 8192 when it runs. Many modern linux systems built to 
+  host large applications such as Presto are expected to support much larger 
+  values, on the order of 32k open files descriptors and above, to ensure that
+  these limits are never hit. These limits are set in 
```

Comment:
It should be clear that this is an example if Presto is running as the `presto` user. Also, this may be Linux distribution specific and depend on how Presto is run. For example, in Kubernetes, the limits are likely set in an entirely different way.

@jvanzyl any suggestions here?

Reply:
Standard EKS now has high ulimits ([issue 206](https://github.com/awslabs/amazon-eks-ami/pull/206)), but modifying them is difficult. In an AWS Fargate EKS cluster, they are set to 1024 and cannot be modified.

Reply:
@electrum - was this concern addressed by the Note?